archinstall/lib/args.py

```diff
@@ -72,7 +72,7 @@ class ArchConfig:
 	ntp: bool = True
 	packages: list[str] = field(default_factory=list)
 	parallel_downloads: int = 0
-	swap: bool = True
+	swap: ZramConfiguration = field(default_factory=lambda: ZramConfiguration(enabled=True))
```

Comment:
No need to use `field` here as that was only for the `dict`, you can assign the `ZramConfiguration(true)` directly